app/src/main/java/de/danoeh/antennapod/fragment/AllEpisodesFragment.java

```diff
@@ -81,7 +81,7 @@
     private boolean itemsLoaded = false;
     private boolean viewsCreated = false;
 
-    private AtomicReference<MainActivity> activity = new AtomicReference<MainActivity>();
+    private AtomicReference<MainActivity> activity = new AtomicReference<>();
```

Comment:
From what I could understand, all it does is prevent `onFragmentLoaded()` from being called after `resetViewState()` (or `onDestroyView()`) has been invoked. Of course, there are ways to do it without any risk of leaks, and the already existing boolean viewsCreated does that same job. One might wonder though if we need to care about synchronization, because there's a point where `onFragmentLoaded()` is called asynchronously from the UI thread.